Automattic/wp-calypso
# A4A: adjust styles to make slider in marketplace sticky

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/334

## Proposed Changes
Adjust some styles to make quantity slider always visible to users. This is not best solution, but given a tight timeline we can start with this and then adjust design and code to have a proper solution.
<img width="1707" alt="Screenshot 2024-04-22 at 8 58 13 PM" src="https://github.com/Automattic/wp-calypso/assets/60262784/0b9fffd6-665d-4288-9a40-b05d6eff21aa">


## Testing Instructions



- Use the live link below and navigate to `/marketplace/products`.
- Check the slider is sticky and always visible to users. 
- Check other elements look good

## Pre-merge Checklist
